src/main/java/net/nikr/eve/jeveasset/gui/tabs/values/ValueTableFormat.java

```diff
@@ -217,6 +217,34 @@ public String getColumnName() {
 		public Object getColumnValue(final Value from) {
 			return from.getBestModuleValue();
 		}
+	},
+	CURRENT_SHIP(String.class, GlazedLists.comparableComparator()) {
+		@Override
+		public String getColumnName() { return TabsValues.get().columnCurrentShip(); }

```

Comment:
Micro request: Formatting should not be contracted, as it's harder to read.
I prefer the full'ish format;
```
		@Override
		public String getColumnName() {
			return TabsIndustrySlots.get().columnCurrentShip();
		}
		@Override
		public Object getColumnValue(final IndustrySlot from) {
			return from.getCurrentShip();
		}
```

Reply:
I'm blaming the IDE for that. I though it was just collapsed in the UI. I agree that expanded is better.